hilla-react/versions.json

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
does hilla have some logic to parse this file?
for example: 
- replacing the {{version}} to a version number
- keep the version inside to be updated